Description: Forces GTK integration so the GTK file picker is always used.

diff --git a/Telegram/SourceFiles/platform/linux/specific_linux.cpp b/Telegram/SourceFiles/platform/linux/specific_linux.cpp
index 340afda80..d94f5cb03 100644
--- a/Telegram/SourceFiles/platform/linux/specific_linux.cpp
+++ b/Telegram/SourceFiles/platform/linux/specific_linux.cpp
@@ -310,6 +310,8 @@ bool UseGtkIntegration() {
 }
 
 bool IsGtkIntegrationForced() {
+	return true;
+
 #ifndef TDESKTOP_DISABLE_GTK_INTEGRATION
 	static const auto Result = [&] {
 		return PlatformThemes.contains(qstr("gtk3"), Qt::CaseInsensitive)
